services/web-app/components/utils/useOutsideClick.js

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Can this go in `/services/web-app/utils` directory, renamed to `use-outside-click.js` to follow naming conventions that you'll see once you pull in latest from main branch?